### ISSUE TITLE
[IMPROVEMENTS]: Snow fall speed should be increased

### ISSUE THREAD

**Issue by LuckFire** [typical-developers/oaklands-feedback#172]
### Roblox Username

coltbarrett112

### What do you think can have quality-of-life improvements?

The speed of snow should be increased.

### How do you think things can be improved?

For walking and running, the rate of snow is fine.

https://github.com/typical-developers/oaklands-feedback/assets/39076891/5d76920d-c731-4ece-a6ec-30904f96d692

However, for all vehicles -- using the slowest one obtainable to show it can happen with all of them -- the snow rate is a bit too slow and ruins the immersion while driving.

https://github.com/typical-developers/oaklands-feedback/assets/39076891/333d89ea-29a9-49db-8ba7-cf198bdbfee9

**Comment by LuckFire** [comment]
After some personal investigation, it seems like it'll be difficult / impossible to actually get this working in a way that looks better than it does currently.

The reason the snow looks off is due to the spawn rate when moving. My thought process was to have a grid of parts to create a "buffer" of snow ahead of the player so by the time they reach that point, snow would be there. This did work partially, however, when moving the camera around it would look worse -- from how we do weather -- than it does currently since the snow would stop falling in one area and start in another.

In a nutshell, this would only work if you were moving your camera little to none. Since this seems impossible to change, I have closed this issue request since I've deemed it unreasonable to accomplish.
